Docker.DotNet.BasicAuth.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Docker.DotNet.BasicAuth</id>
+    <version>2.124.3</version>
```

Comment:
We can, like I said, that will be updated to accept the build number as a parameter from the one calling the build (i.e. AppVeyor)